mkt/extensions/indexers.py

```diff
@@ -113,7 +120,9 @@ def extract_document(cls, pk=None, obj=None):
         else:
             doc['latest_public_version'] = None
         doc['name_sort'] = unicode(obj.name).lower()
-        doc['guid'] = unicode(obj.uuid)
+        # Find the first reviewed date (used in sort).
```

Comment:
Yeah. I suspect we did that in order to avoid having developers constantly push new updates to stay at the top of the new page (they'd need reviews, but still)